packages/npm/send/ios/Send.swift

```diff
@@ -18,101 +8,46 @@ private func bodyIsUTF8(contentTypeHeader: String?, utf8ContentTypes: [String])
-    }
-}
+extension SendError: Error {}
 
-struct Request: Decodable {
-    let baseURL: String
-    let path: String
-    let queryParameters: [String: String]
-    let headerParameters: [String: String]
-    let method: String
-    @RequiredKey var body: String?
-    let utf8ContentTypes: [String]
-    
+extension Request {
     var url: Result<URL, SendError> {
         guard var urlComponents = URLComponents(string: baseURL) else {
-            return .failure(.invalidRequestBaseURL)
+            return .failure(SendError(code: .invalidRequestBaseUrl, message: "Your base URL is not valid."))
         }
         urlComponents.path = path
         
-        if !queryParameters.isEmpty {
-            urlComponents.queryItems = queryParameters.map { key, value in
+        if !query.parameters.isEmpty {
+            urlComponents.queryItems = query.parameters.map { key, value in
                 URLQueryItem(name: key, value: value)
             }
         }
         
         guard let url = urlComponents.url else {
-            return .failure(.invalidRequestPathOrQueryParameters)
+            return .failure(SendError(code: .invalidRequestPathOrQueryParameters, message: "Your path or query parameters is not valid."))
         }
         return .success(url)
     }
     
     var urlRequest: Result<URLRequest, SendError> {
         return url.flatMap { url in
             var urlRequest = URLRequest(url: url)
-            urlRequest.httpMethod = method
+            urlRequest.httpMethod = method.stringValue
             
-            for (key, value) in headerParameters {
+            for (key, value) in header.parameters {
                 urlRequest.setValue(value, forHTTPHeaderField: key)
             }
             
             if let body {
-                let contentTypeHeader = headerParameters.first(where: { $0.key.caseInsensitiveCompare("Content-Type") == .orderedSame })?.value
+                let contentTypeHeader = header.parameters.first(where: { $0.key.caseInsensitiveCompare("Content-Type") == .orderedSame })?.value
                 if bodyIsUTF8(contentTypeHeader: contentTypeHeader, utf8ContentTypes: utf8ContentTypes) {
                     guard let utf8Body = body.data(using: .utf8) else {
-                        return .failure(.nonUTF8RequestBody)
+                        return .failure(SendError(code: .nonUtf8RequestBody, message: "Your request headers specify a Content-Type included in `utf8ContentTypes`, but your request body is not a UTF8-encoded string."))
                     }
                     urlRequest.httpBody = utf8Body
                 } else {
                     guard let base64Body = Data(base64Encoded: body) else {
-                        return .failure(.nonBase64RequestBody)
+                        return .failure(SendError(code: .nonUtf8RequestBody, message: "Your request headers specify a Content-Type NOT included in `utf8ContentTypes`, but your request body is not a base64-encoded string."))
```

Comment:
```suggestion
                        return .failure(SendError(code: .nonBase64RequestBody, message: "Your request headers specify a Content-Type NOT included in `utf8ContentTypes`, but your request body is not a base64-encoded string."))
```

Regression from nitro upgrade commit